pkg/kev/converter/kubernetes/project_service_test.go

```diff
@@ -868,23 +866,22 @@ var _ = Describe("ProjectService", func() {
 
 		Context("for invalid image pull policy", func() {
 			policy := "invalid-policy-name"
+			extensions := make(map[string]interface{})
 
-			BeforeEach(func() {
-				labels = composego.Labels{
-					config.LabelWorkloadImagePullPolicy: policy,
-				}
+			JustBeforeEach(func() {
+				k8sconf.Workload.ImagePull.Policy = policy
```

Comment:
How about moving this particular line to the BeforeEach block as you did higher up in line 853?